openstack/messaging/v2/claims/requests.go

```diff
@@ -0,0 +1,73 @@
+package claims
+
+import (
+	"net/http"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+// CreateOptsBuilder Builder.
+type CreateOptsBuilder interface {
+	ToClaimCreateRequest() (map[string]interface{}, string, error)
+}
+
+// CreateOpts params to be used with Create.
+type CreateOpts struct {
+	// The name of the queue to claim messages from.
+	QueueName string `json:"queue_name" required:"true"`
+
+	// Sets the TTL for the claim. When the claim expires un-deleted messages will be able to be claimed again.
+	TTL int `json:"ttl,omitempty"`
```

Comment:
To make sure: 0 is not a valid TTL? If it is, this should be `*int`

Reply:
"The ttl value must be between 60 and 43200 seconds (12 hours). "